torch/nn/quantized/modules/linear.py

```diff
@@ -36,7 +36,8 @@ def forward(self, X):
 
     @staticmethod
     def from_float(mod):
-        return Quantize(mod.qparams[0].item(), mod.qparams[1].item(), torch.quint8)
+        qparams = mod.observer.calculate_qparams()
```

Comment:
better error messaging? if `.observer` is not present - produce an error that the module hasn't been instrumented